packages/babel-core/src/index.js

```diff
@@ -0,0 +1,69 @@
+import isFunction from "lodash/isFunction";
+import fs from "fs";
+
+export { default as File } from "./transformation/file";
+export { default as options } from "./transformation/file/options/config";
+export { default as buildExternalHelpers } from "./tools/build-external-helpers";
+export { default as template } from "babel-template";
+export { default as resolvePlugin } from "./helpers/resolve-plugin";
+export { default as resolvePreset } from "./helpers/resolve-preset";
+export { version } from "../package";
+
+import * as util from "util";
```

Comment:
Also, not sure the "move" took hold, as it looks like `api/node.js` still exists?

Doesn't look like we need the `api` folder anymore either.